### PR TITLE
fix EOL in gremlin-server.sh

### PR DESCRIPTION
**Pull request by vkagamlyk**
fix EOL in gremlin-server.sh for gremlin-go tests

### CODE REVIEW

**Review by kenhuuu — APPROVED** [review]

File: docker/gremlin-test-server/Dockerfile [hunk 1]
```diff
@@ -28,7 +28,7 @@ RUN chmod 755 /opt/docker-entrypoint.sh
 # Installing dos2unix to avoid errors in running the entrypoint script on Windows machines where their
 # carriage return is \r\n instead of the \n needed for linux/unix containers
 RUN apk update && apk add dos2unix
-RUN dos2unix /opt/docker-entrypoint.sh && apk del dos2unix
+RUN dos2unix /opt/docker-entrypoint.sh /opt/gremlin-server/bin/gremlin-server.sh && apk del dos2unix
```
Comment (L0Lmaker) [inline docker/gremlin-test-server/Dockerfile:31]: What exactly does this change fix?

Comment (vkagamlyk) [reply docker/gremlin-test-server/Dockerfile:31]: windows-like EOF:
`/opt/docker-entrypoint.sh: /opt/gremlin-server/bin/gremlin-server.sh: /bin/bash^M: bad interpreter: No such file or directory`

Comment (L0Lmaker) [reply docker/gremlin-test-server/Dockerfile:31]: oh gotcha

**Review by L0Lmaker — APPROVED** [review]
LGTM

**Review by xiazcy — APPROVED** [review]

**Pull request merged**